app/jest.config.js

```diff
@@ -4,7 +4,9 @@ module.exports = {
   collectCoverageFrom: ['src/**/*.js', '!frontend/**/*.*'],
   moduleFileExtensions: ['js', 'json'],
   moduleNameMapper: {
-    '^@/(.*)$': '<rootDir>/src/$1'
+    '^@/(.*)$': '<rootDir>/src/$1',
+    // axios now built as ES Module (see: https://github.com/axios/axios/issues/5101)
+    '^axios$': require.resolve('axios'),
```

Comment:
> Try upgrading jest to latest (>29.2.x), it worked for me without having to change jest config.

If we are doing a dependency update, we should consider updating more than just axios so that hopefully we don't need to be inserting in exception mappers like this to circumvent the CJS -> EJS issue. Also note that since CHES is a very old product, it may not be using the `jest.mock(axios)` pattern to outright shuttle away the compilation issues that we do in newer projects.

Reply:
updated to jest 29.3.1